builder/comp-builder.nix

```diff
@@ -91,12 +91,18 @@ let
       "$(cat ${configFiles}/configure-flags)"
     ] ++ commonConfigureFlags);
 
+  # From nixpkgs 20.09, the pkg-config exe has a prefix matching the ghc one
+  pkgConfigHasPrefix = !(lib.strings.hasPrefix "19."   lib.nixpkgsVersion
+                      || lib.strings.hasPrefix "20.03" lib.nixpkgsVersion);
```

Comment:
Can't we do one of those version comparison thingys we do elsewhere, e.g. `versionLessThan "20.09"`?